### DISCUSSION TITLE
FS2020 mod and LITLLENAVMAP

### DISCUSSION THREAD

**Discussion by ecaport** [albar965/littlenavmap#795] [Ideas]
Hello, I'm Paolo, I am developing airports for MSFS2020, I use LittleNavMap as a design aid.
I realized that LNM reads both mods and navs together, it follows that the same airport on fs nav and on the mod in community adds up. You should give precedence to mods like the MSFS2020 simulator does.
Otherwise a confusion arises because I find myself parking and rest, made by ASOBO and my parking and rest made on the mod that you see both.
Thanks for the attention
Paolo by Italy

**Comment by albar965** [comment]
Hi Paolo,

not sure if I understand. Can you give me more details and probably example files?

LNM prioritizes airport addons before stock but not navaids. Reading airports is already an insane task due to the over-complicated update system of FSX/P3D/MSFS. It will be some time until I can implement this for navaids too (if at all).

Alex

**Comment by ecaport** [comment]
Hi Alex. Thank you for your answer.
I come to an example:
The Catania LICC airport in Italy exists on the MSFS200, now I have to
modify this airport because it is different from reality.
So I created a mod through the SDK section of the MSFS2020 of the new LICC
airport, in this mod I only kept the runway, I deleted through the
multi-command delete, which exists in the MSFS2020 SDK, all the taxiways,
the aprons. the ILS.
I ran the MSFS2020 simulator with this mod and the simulator sees my job
right, that is, it only sees the runway but no longer sees the parking lots
and not even the taxiways, this is because I modified the LICC airport
through the new mod that I installed in the specific Community directory of
MSFS2020.
But in LNM if I update the scenarios I see the airport as it is on the
MSFS2020 simulator, as originally, before I made the mod., I review the
parking lots and the taxiways as it was before making the mod.
This is why I believe that LNM does not prioritize mods (addons).
Example: if MSFS2020 contains LICC inside, and if you insert a mod in the
COMMUNITY that contains LICC (ICAO of Catania),
LNM shows them both.
Attached is a screen showing that LICC still has all the taxis and parking
lots, while on the MSFS2020 simulator neither parking nor taxiways appear.
A thousand thanks

Il giorno dom 10 ott 2021 alle ore 17:15 Alexander Barthel <
***@***.***> ha scritto:

**Comment by albar965** [comment]
Not sure. Might be an issue in LNM.
Can you send me the BGL file(s) per email? 
https://albar965.github.io/contact.html
Alex

**Comment by ecaport** [comment]
Hi Alex, I am sending you the same project in which, inside you will find
the PACKAGE directory that contains the mod to be inserted in the MSFS2020
sim with the bgl
A thousand thanks
Link for download
https://drive.google.com/file/d/1w0wl72pqdTFI9CazjL6GxsrK86-xoMGA/view?usp=sharing

Il giorno lun 11 ott 2021 alle ore 14:22 Alexander Barthel <
***@***.***> ha scritto:

**Comment by albar965** [comment]
Thank you for the files. Got them.

**Comment by albar965** [comment]
I created an issue #798 related to this.
I'm currently working on other tasks in LNM now and it might take some time until I get to #798.

**Comment by ecaport** [comment]
ok , grazie Alex, buon lavoro, complimenti ancora per LNM.
Paolo

Il mar 12 ott 2021, 17:40 Alexander Barthel ***@***.***> ha
scritto:

**Comment by ecaport** [comment]
Hi Alex, I found out that if I put a parking lot and a taxiway in my LICC
airport then LNM will update right to this airport, if I don't put anything
LNM reads the LICC from the fsnav, while if on the LICC mod I enter 1
parking and new LNM taxi works just right.
This is to update the state of affairs.
Hello
Paul

Il giorno mar 12 ott 2021 alle ore 17:40 Alexander Barthel <
***@***.***> ha scritto:

**Comment by albar965** [comment]
Thanks for the feedback. There might be an issue in LNM decoding of delete records. Not sure but it happened already due to a lot of undocumented corner cases.

**Comment by ecaport** [comment]
Thanks for your always prompt reply, good job.
Time to make another donation.
Hello
Paolo

Il giorno dom 17 ott 2021 alle ore 18:11 Alexander Barthel <
***@***.***> ha scritto:

**Reply by albar965** [reply]
Thank you for your donation! 

The fix for this issue did not make it into 2.6.17, sorry. I was in a bit of a hurry to release before SU6.

Alex

**Comment by ecaport** [comment]
Dear Alex , no problem.
Ciao

Il lun 18 ott 2021, 21:42 Alexander Barthel ***@***.***> ha
scritto:
